products/spanner/inspec.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Inspec::Config
+legacy_name: appengine
+overrides: !ruby/object:Overrides::ResourceOverrides
+  ApplicationUrlDispatchRules: !ruby/object:Overrides::Inspec::ResourceOverride
```

Comment:
Looks like this file got mixed up with `products/appengine/inspec.yaml`. These are all references to app engine resources

Reply:
ya.. it got mixed up with so many open file. I have corrected this.